Source/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -179,13 +239,15 @@ static void CollectAccessibilityElementsForContainer(ASDisplayNode *container, U
 
   if (AS_AVAILABLE_IOS_TVOS(11, 11)) {
     NSArray *attributedLabels = [labeledNodes valueForKey:@"accessibilityAttributedLabel"];
-    NSMutableAttributedString *attributedLabel = [NSMutableAttributedString new];
-    [attributedLabels enumerateObjectsUsingBlock:^(id  _Nonnull obj, NSUInteger idx, BOOL * _Nonnull stop) {
+    NSMutableAttributedString *attributedLabel = [[NSMutableAttributedString alloc] init];
+    [attributedLabel beginEditing];
+    [attributedLabels enumerateObjectsUsingBlock:^(NSAttributedString * _Nonnull obj, NSUInteger idx, BOOL * _Nonnull stop) {
```

Comment:
While you're in here, remove stray `_Nonnull`s.

Reply:
Done